README.md

```diff
@@ -67,6 +67,7 @@ Besides the above, there are many possible options within `config.yml` for confi
     - `resign_moves`: The evaluation has to be less than or equal to `resign_score` for `resign_moves` amount of moves for the bot to resign.
     - `offer_draw_enabled`: Whether the bot is allowed to offer/accept draw based on the evaluation.
     - `offer_draw_score`: The absolute value of the engine evaluation has to be less than or equal to `offer_draw_score` for the bot to offer/accept draw.
+    - `offer_draw_egtb_score`: The absolute value of the wdl score from the online egtb has to be less than or equal to `offer_draw_egtb_score` for the bot to offer/accept draw.
```

Comment:
Since wdl is only ever -2, -1, 0, 1, or 2, should this option be a true/false configuration? That is, if the egtb returns a 0 score, then offer a draw?

Reply:
In the last commit it became true/false, but I am not sure if this is what you meant.

Reply:
if you look at this function:

```python
def score_to_wdl(score):
    if score < -20000:
        return -2
    elif score < 0:
        return -1
    elif score == 0:
        return 0
    elif score <= 20000:
        return 1
    else:
        return 2
```

A wdl of -1 can mean anything from slight disadvantage to very-losing-but-the-route-to-checkmate-is-slightly-too-long-to-prove (and similarly for wdl of 1). This is too wide a range to be offering draws. If I were writing this, only wdl = 0 would result in a draw offer and the option would be called `offer_draw_for_egtb_zero`.

```python
if draw_or_resign_cfg.get('offer_draw_enabled', False) and draw_or_resign_cfg.get('offer_draw_for_egtb_zero', False) and wdl == 0:
    offer_draw = True
```
Or, so the line isn't so long
```python
can_offer_draw = draw_or_resign_cfg.get('offer_draw_enabled', False)
offer_egtb_draw = draw_or_resign_cfg.get('offer_draw_for_egtb_zero', False)
if can_offer_draw and offer_egtb_draw and wdl == 0:
    offer_draw = True
```
I am open to arguments that your way is better.

Reply:
I added `offer_draw_for_egtb_zero` and `resign_for_egtb_minus_two`. Now the bot won't offer draws in positions with wdl of 1 or -1. A wdl of -1 means that the position is lost but the 50 move rule makes it a draw. A score of 20000 by the chessdb egtb isn't the same as a score of 20000 by a chess engine. The positions with wdl of 1 or -1 are converted to score by chessdb with `score = wdl * 20000 - wdl * dtz`, while in chess engines it means that the position is winning but I haven't found a mate yet. Chess engines will usually have a score of about 400 in positions with wdl of 1. If two bots have the online_egtb enabled or have the TB files on their computer, the bots will draw even though chessdb returns a score of 20000.